src/components/Table/GridTable.stories.tsx

```diff
@@ -1849,3 +1854,146 @@ export function Headers() {
     </div>
   );
 }
+
+/**
+ * Shows how drag & drop reordering can be implemented with GridTable drag events
+ */
+export function DraggableRows() {
```

Comment:
Praise: I popped this open in storybook and it looks slick!

Reply:
Seems like the only thing this doesn't do is scroll the page/table when I get to the top and bottom, but otherwise it seems pretty functionally complete? Very nice! Pretty great that it's just native events too, and not bringing in a new library. Kinda assumed we'd just need that, but this looks good!

Reply:
It was scrolling for me when dragging to top/bottom, thought there seems to be a small window of where you need to hold the row for it to work 🤔 

praise: Looks great @bsholmes

Reply:
Yeah the scrolling behavior isn't ideal, and is relying on the ```dataTransfer.effectAllowed = "move"``` stuff so isn't tweakable. We may need to implement our own scrolling behavior, at least on mobile.